src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,166 @@
+import { Placement } from "@popperjs/core"
+import {
+  CSSProperties,
+  KeyboardEvent,
+  ReactNode,
+  useEffect,
+  useRef,
+  useState,
+} from "react"
+import { usePopper } from "react-popper"
+import { Transition, TransitionStatus } from "react-transition-group"
+import classes from "./Tooltip.module.css"
+
+interface TooltipProps {
+  children: ReactNode
+  label: string
+  ariaDescribedBy?: string
```

Comment:
Would it be worth extending an extensive list of aria properties or is `described by` the only one that makes sense here?

Reply:
I was going off of this documentation here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role

Feel free to poke around and see if I'm missing anything. Happy to expose more aria properties if we feel that is best.

Reply:
It looks like `described by` might be all that's needed :+1:

My question comes from the interfaces that we have been using for components here, more often than not we just expose one / a few aria attributes but I wonder if we could extend `AriaAttributes` and just give the user the full list of aria attributes. 

But then users would need to know which ones to use, if we explicitly expose them that's kind of a hint of what attributes they should be using.

All this rambling to come to the conclusion that it's probably best to expose a subset of aria attributes instead of the whole thing.

Reply:
Could we `<Pick>` the relevant attributes from `AriaAttributes` instead of retyping them?

Reply:
I'm good with that. Like this? `interface TooltipProps extends Pick<AriaAttributes, "aria-describedby">`

Reply:
then in props `  ...aria`

and finally implemented like so ` aria-describedby={aria["aria-describedby"]}`

Reply:
Or I guess at that point we can just spread the relevant attributes.

Reply:
@thispastwinter yeah I'd think so. Not totally clear on how the implementation would look like, just a thought.